apps/website/.vuepress/theme/global-components/DocInset.vue

```diff
@@ -25,6 +25,18 @@ export default {
   justify-content: center;
   align-items: center;
   flex-direction: column;
+
+  @media (max-width: 1024px) {
+    width: 100%;
+    overflow: auto;
+
+    header {
```

Comment:
@hippee-lee I am open to moving this one to another component or if we have some kind of global style file(didn't find one)